lib/daru/io/io.rb

```diff
@@ -220,6 +232,41 @@ def from_csv_hash(path, opts)
 
         headers.each_with_index.map { |h, i| [h, csv_as_arrays[i]] }.to_h
       end
+
+      # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
```

Comment:
Unacceptable, sorry :( You should try to write better code, not disable everything you dislike.